package.json

```diff
@@ -4,43 +4,125 @@
+    "lint": "run-s lint:js lint:styles lint:php",
+    "lint:js": "wds-format-js && wds-lint-js",
+    "lint:php": "composer install --quiet && composer run lint",
+    "lint:styles": "wds-format-styles && wds-lint-style",
+    "packages-update": "wds-packages-update",
     "postinstall": "npm run build",
-    "lint": "npm run lint:js && npm run lint:style",
-    "lint:style": "wp-scripts lint-style",
-    "theme": "run-s pot icons",
-    "pot": "wp i18n make-pot . build/languages/_s.pot --exclude=node_modules,vendor,build",
-    "icons": "svgstore build/images/icons/*.svg --inline -o build/images/icons/sprite.svg"
+    "preinstall": "$(echo node --version) | grep -sq $(cat .nvmrc) || ( echo \"\\n\\n\\n\\t!!! Please use Node version $(cat .nvmrc), you are using version $(node --version), some things may fail to work. !!!\\n\\n\\n\\n\" )",
+    "start": "wds-start",
+    "theme": "run-s theme:pot theme:icons",
+    "theme:icons": "svgstore build/images/icons/*.svg --inline -o build/images/icons/sprite.svg",
+    "theme:pot": "wp i18n make-pot . build/languages/_s.pot --exclude=node_modules,vendor,build",
+    "test": "npm install && run-s lint build && [[ -d build ]] && [[ -d src ]] && [[ -d inc ]] && [[ -d template-parts ]] && [[ -f style.css ]] && [[ -f webpack.config.js ]]"
   },
   "keywords": [],
-  "author": "",
+  "author": "WebDevStudios",
   "license": "ISC",
-  "dependencies": {
-    "npm-run-all": "^4.1.5"
-  },
+  "dependencies": {},
   "devDependencies": {
-    "@webdevstudios/css-coding-standards": "^1.0.1",
-    "@webdevstudios/js-coding-standards": "^1.1.1",
-    "@wordpress/scripts": "^12.1.1",
+    "@webdevstudios/js-coding-standards": "~1.3.0",
+    "@webdevstudios/npm-scripts": "~1.0.1",
+    "@wordpress/scripts": "~12.3.0",
     "copy-webpack-plugin": "^6.1.1",
     "cssnano": "^4.1.10",
+    "husky": "~4.3.0",
     "imagemin-webpack-plugin": "^2.4.2",
+    "lint-staged": "~10.4.1",
+    "npm-run-all": "~4.1.5",
+    "prettier": "~2.1.2",
+    "run-s": "0.0.0",
+    "stylelint": "~13.7.2",
+    "stylelint-config-wordpress": "~17.0.0",
     "svgstore-cli": "^2.0.0",
     "tailwindcss": "^1.8.0"
   },
   "eslintConfig": {
-    "extends": [
-      "plugin:@wordpress/eslint-plugin/esnext",
-      "plugin:@wordpress/eslint-plugin/react",
-      "@webdevstudios/js-coding-standards"
-    ],
+    "extends": "@webdevstudios/js-coding-standards",
     "ignorePatterns": [
-      "build/**",
-      "vendor/**"
+      "**/*.css",
+      "**/*.json",
+      "**/*.lock",
+      "**/*.php",
+      "**/*.png",
+      "**/*.scss",
+      "**/*.svg",
+      "*.min.js",
+      ".*dist.js",
+      ".*rc.js",
+      "./.*.js",
+      ".cache/**/*",
+      ".git*",
+      "build/**/*",
+      "cache/**/*",
+      "dist/**/*",
+      "node_modules/**/*",
+      "vendor/**/*"
+    ]
+  },
+  "stylelint": {
+    "extends": "stylelint-config-wordpress/scss",
+    "rules": {
```

Comment:
We do not use @webdevstudios/css-coding-standards, right now wd_s uses a custom config. But soon we will port this over to css-coding-standards and remove this. Added a few extra rules @coreymcollins might notice to make sure things work harmoniously until then.